### DISCUSSION TITLE
WebSockets

### DISCUSSION THREAD

**Discussion by AuditConsult** [InnovateAsterisk/Browser-Phone#456] [Q&A]
I have a quite number of issues,
1. I have configured my tlsbindaddr to port 80899 (https) and port 8088 for http in http.conf. I can access httpstatus page via http on port 8088 and not on https on 8089 but I have configured my certificates correctly.
2. when am triying to access 156.36.148.198:8088/ws it says upgrade required I don't know whats going on.
3. my browser phone can't access the microphone and camera when the system in on the server am using ubuntu server. only access the camera when am using my pc
![Screenshot 2023-05-17 at 10 25 41 PM](https://github.com/InnovateAsterisk/Browser-Phone/assets/115872230/0b60b99e-aca5-4857-a777-41621ef2f6e0)

**Comment by InnovateAsterisk** [comment]
You appear to be connecting the browser to the web socket url. You would navigate the browser to the https home page or a location that is hosting the html and JavaScript. Then when the phone opens, in the account details you specify the web socket details (156.36.148.198:8088/ws). You will probably not use the http port.
